src/components/Tabs/Tabs.jsx

```diff
@@ -2,0 +1,31 @@
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected, setActiveTabId }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={cn({ 'is-active': activeTabId === tab.id })}
+              data-cy="Tab"
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => setActiveTabId(tab.id)}
+              >
+                {tab.title}
+              </a>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {onTabSelected ? onTabSelected.content : 'No content'}
```

Comment:
The `onTabSelected` variable is used as if it contains a tab object. Ensure that `onTabSelected` is a function that determines the content based on the active tab.